docs/pt_BR/build.md

```diff
@@ -0,0 +1,75 @@
+# Build
+
+Esta página também está disponível em [Japônes](https://github.com/BoostIO/Boostnote/blob/master/docs/jp/build.md), [Coreano](https://github.com/BoostIO/Boostnote/blob/master/docs/ko/build.md), [Russo](https://github.com/BoostIO/Boostnote/blob/master/docs/ru/build.md), [Chinês simplificado](https://github.com/BoostIO/Boostnote/blob/master/docs/zh_CN/build.md), [Francês](https://github.com/BoostIO/Boostnote/blob/master/docs/fr/build.md) e [Alemão](https://github.com/BoostIO/Boostnote/blob/master/docs/de/build.md).
+
+## Ambiente
+
+- npm: 6.x
+- node: 8.x
+
+## Desenvolvimento
+
+Nós usamos o Webpack HMR para desenvolver o Boostnote.
+Executando os seguintes comandos, no diretório raiz do projeto, irá iniciar o Boostnote com as configurações padrão.
```

Comment:
I guess `será iniciado` is better than `irá iniciar` in this context.
You also can use `Ao executar os seguintes comandos no diretório raiz do projeto, o Boostnote será iniciado com as configurações padrão.`